contents/blog/ceo-diary-3.md

```diff
@@ -0,0 +1,105 @@
+* Regular scans of her seeing eye under general anaesthetic.
+
+It feels pretty weird how much of a relationship we're starting to build with the staff at the various hospitals involved. We've got their cell numbers, and they tell us all the gossip on results or how to get through the system. 
+
+## So, the board meeting
+
+Cancer has taken an eye; my way of saying screw you to it is that I'm not going to let it take away or reduce the opportunity at PostHog for anyone - directly or indirectly (including my family) involved.
+
+We've had tremendous progress. Our goal for August was to get five reference customers for our paid self hosted product. We've now gone way past this and are at around 10, with around 100 paying customers on cloud, and a further 8,000 ish companies in total deployed.
+
+There are two modes for product market fit:
+
+* 0 to 1
+* 1 to many
+
+This means we feel that:
+
+* Open source project = 1 to many mode
+* PostHog Cloud = 1 to many mode (although we are focused on self hosted)
+* PostHog Scale = 1 to many mode
+
+## Next up
+
+We now need to:
+
+* More in: Accelerate growth of the open source product and the Scale product.
+* More out: Get higher order values. We are working on our Enterprise product. This has a longer cycle since larger organizations are slower moving.
+
+These two things together we believe will compound.
+
+### Getting more in
+
+This is an unknown unknown. So the only way to figure it out is to get started making our own luck.
+
+We've been blessed with mainly organic growth to date. The advantage is that this product-led channel is very leveraged - we can double down on our product and we will grow faster (more word of mouth growth, and better retention). However, we can't create wild product improvements overnight - we need to push on more channels than this to accelerate.
+
+As we created our reference customers, we've realized what the most successful users of PostHog [have in common](/handbook/strategy/strategy).
+
+First, we need to create a scorecard. Long term revenue is the _output_ but we want to look upstream for a metric that is more controllable, with a faster feedback loop.
+
+We are creating a simple algorithm that scores each company that deploys (and is opted in). This is based on their usage and company data. We're doing this by sending [PostHog data to Hubspot](https://github.com/PostHog/hubspot-plugin) and using their lead scoring. This lets us figure out if we're getting high quality signups. Once we have this, we can figure out if our work in marketing and developer relations is having an impact.
+
+### More out
+
+We've some huge fans using us in production in massive enterprises. It stands to reason - they care greatly about data control, and they're able to deploy PostHog the day they find it since all data stays in their infrastructure, and they can start for free.
```

Comment:
```suggestion
We've some huge fans using us in massive enterprises. It stands to reason - they care greatly about data control, and they're able to deploy PostHog the day they find it since all data stays in their infrastructure, and they can start for free.
```